_dashboards/visualize/maps-stats-api.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: default
+title: Maps Stats API
+nav_order: 20
+grand_parent: Building data visualizations
+parent: Using coordinate and region maps 
+has_children: false
+---
+
+# Maps Stats API
+Introduced 2.7
+{: .label .label-purple }
+
+When you create and save a [map]({{site.url}}{{site.baseurl}}/dashboards/visualize/maps/) in OpenSearch Dashboards, the map becomes a saved object of type `map`. The Maps Stats API provides information about such saved objects in OpenSearch Dashboards. 
+
+#### Example request
+
+You can access the Maps Stats API by providing its URL address in the following format:
+
+```
+<opensearch-dashboards-endpoint-address>/api/maps-dashboards/stats
+```
+
+The OpenSearch Dashboards endpoint address may contain a port number if it is specified in the OpenSearch configuration file. The specific URL format depends on the type of OpenSearch deployment and the network environment in which it is hosted.
+{: .note}  
+
+You can query the endpoint in two ways:
+  
+  - By accessing the endpoint address (for example`http://localhost:5601/api/maps-dashboards/stats`) in a browser
+
+  - By using the `curl` command in the terminal:
+    ```bash
+    curl -X GET http://localhost:5601/api/maps-dashboards/stats
+    ```
+    {% include copy.html %}
+
+#### Example response
+
+The following is the response for the preceding request:
+
+```json
+{
+   "maps_total":4,  
+   "layers_filters_total":4, 
+   "layers_total":{ 
+      "opensearch_vector_tile_map":2, 
+      "documents":7, 
+      "wms":1, 
+      "tms":2 
+   },
+   "maps_list":[
+      {
+         "id":"88a24e6c-0216-4f76-8bc7-c8db6c8705da", 
+         "layers_filters_total":4,
+         "layers_total":{
+            "opensearch_vector_tile_map":1,
+            "documents":3,
+            "wms":0,
+            "tms":0
+         }
+      },
+      {
+         "id":"4ce3fe50-d309-11ed-a958-770756e00bcd",
+         "layers_filters_total":0,
+         "layers_total":{
+            "opensearch_vector_tile_map":0,
+            "documents":2,
+            "wms":0,
+            "tms":1
+         }
+      },
+      {
+         "id":"af5d3b90-d30a-11ed-a605-f7ad7bc98642",
+         "layers_filters_total":0,
+         "layers_total":{
+            "opensearch_vector_tile_map":1,
+            "documents":1,
+            "wms":0,
+            "tms":1
+         }
+      },
+      {
+         "id":"5ca1ec10-d30b-11ed-a042-93d8ff0f09ee",
+         "layers_filters_total":0,
+         "layers_total":{
+            "opensearch_vector_tile_map":0,
+            "documents":1,
+            "wms":1,
+            "tms":0
+         }
+      }
+   ]
+}
+```
+
+## Response fields
+
+The response contains statistics for the following layer types:
+
+- Basemaps: Either an OpenSearch default map or custom base layer maps you created.
```

Comment:
```suggestion
- Basemaps: Either an OpenSearch default map or custom base layer map you created.
```

Reply:
There may be more than one base layer map, so I'll keep plural here :)